yt/data_objects/tests/test_particle_filter.py

```diff
@@ -65,7 +82,7 @@ def was_called():
     # Restore the original warning function
     mylog.warning = warning
 
-def test_particle_filter():
+def test_particle_filter_1():
```

Comment:
Can you give this function and the new one you added below a more descriptive name?

Reply:
done